styles.css

```diff
@@ -27,9 +38,16 @@ aside{
   background:#ccc;
   padding:1em;
   border: 5px solid #aaa;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+
+}
+
 }
 img{
-  max-width:100%;
+  height: 75px;
+  width: 75px;
 }
 
 @media(max-width:600px){
```

Comment:
Don't see any CSS accounting for Question #7. You'll want to use Flexbox's `order` property somewhere in the mobile styling section (`@media(...)`).